core/node/node_framework/src/implementations/layers/web3_api/tx_sink.rs

```diff
@@ -33,9 +33,9 @@ impl WiringLayer for TxSinkLayer {
                 TxSinkResource(Arc::new(MasterPoolSink::new(pool)))
             }
             TxSinkLayer::ProxySink { main_node_url } => {
```

Comment:
I think it makes sense to make `L2Client` a resource and get rid of `main_node_url` field in this variant. @popzxc WDYT?